packages/rustcat/PKGBUILD

```diff
@@ -29,6 +28,7 @@ build() {
 package() {
   cd $pkgname
 
-  install -Dm 755 "target/release/$_binname" "$pkgdir/usr/bin/$_binname"
+  install -Dm 755 "target/release/rcat" -t "$pkgdir/usr/bin/"
```

Comment:
keep the `_binname` variable

Reply:
why?

Reply:
Using a variable instead of hardcoding a string in the PKGBUILD is our standard way to do it (https://github.com/BlackArch/blackarch-pkgbuilds/) even if the variable is used only once.